chainntnfs/neutrinonotify/neutrino.go

```diff
@@ -54,8 +54,8 @@ type NeutrinoNotifier struct {
 	spendClientCounter uint64 // To be used atomically.
 	epochClientCounter uint64 // To be used atomically.
 
-	heightMtx  sync.RWMutex
-	bestHeight uint32
+	bestBlockMtx sync.RWMutex
+	bestBlock    chainntnfs.BlockEpoch
```

Comment:
It doesn't seem like we gain anything from also keeping track of our best hash as we can't rewind until the common ancestor, so this probably isn't needed.